content/pl/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,94 @@
+* Obserwowalność obejmuje nie tylko informacje i metryki z poziomu systemu operacyjnego, ale także poprawność działania samej aplikacji i inne sygnały.
+* Spójność środowiska na etapach rozwoju oprogramowania, testowania i działania w trybie produkcyjnym: działa w ten sam sposób na laptopie i w chmurze.
+* Możliwość przenoszenia pomiędzy systemami operacyjnymi i platformami chmurowymi: Ubuntu, RHEL, CoreOS, on-prem, Google Kubernetes Engine czy gdziekolwiek indziej.
+* Zarządzanie, które w centrum uwagi ma aplikacje: Poziom abstrakcji przeniesiony jest z warstwy systemu operacyjnego działającego na maszynie wirtualnej na poziom działania aplikacji, która działa na systemie operacyjnym używając zasobów logicznych.
+* Luźno powiązane, rozproszone i elastyczne "swobodne" mikro serwisy: Aplikacje podzielone są na mniejsze, niezależne komponenty, które mogą być dynamicznie uruchamiane i zarządzane - nie jest to monolityczny system działający na jednej, dużej maszynie dedykowanej na wyłączność.
+* Izolacja zasobów: wydajność aplikacji możliwa do przewidzenia
+* Wykorzystanie zasobów: wysoka wydajność i upakowanie.
+
+## Do czego potrzebujesz Kubernetesa i jakie są jego możliwości
+
+Kontenery są dobrą metodą na opakowywanie i uruchamianie aplikacji. W środowisku produkcyjnym musisz zarządzać kontenerami, w których działają aplikacje i pilnować, aby nie było żadnych przerw w ich dostępności. Przykładowo, kiedy jeden z kontenerów przestaje działać, inny musi zostać uruchomiony. Nie byłoby prościej, aby takimi działaniami zajmował się jakiś system?
+
+I tu właśnie Kubernetes przychodzi z pomocą! Kubernetes dostarcza środowisko do pracy w niezawodny sposób z systemami rozproszonymi. Kubernetes obsługuje skalowanie aplikacji, przełączanie w sytuacjach awaryjnych, różne scenariusze wdrożeń itp. Przykładowo, Kubernetes w łatwy sposób może zarządzać wdrożeniem nowej wersji oprogramowania zgodnie z metodyką *canary*.
+
+Kubernetes zapewnia:
+
+* **Detekcję nowych serwisów i balansowanie ruchu**  
+Kubernetes może udostępnić kontener używając nazwy DNS lub swojego własnego adresu IP. Jeśli ruch przychodzący do kontenera jest duży, Kubernetes może balansować obciążenie i przekierować ruch sieciowy, aby zapewnić stabilność całej instalacji.
+* **Zarządzanie obsługą składowania danych**  
+Kubernetes umożliwia automatyczne montowanie systemów składowania danych dowolnego typu — lokalnych, od dostawców chmurowych i innych.
+* **Automatyczne instalacje i wycofywanie zmian**  
+Możesz opisać oczekiwany stan instalacji za pomocą Kubernetes, który zajmie się doprowadzeniem w sposób kontrolowany stanu faktycznego do stanu oczekiwanego. Przykładowo, przy pomocy Kubernetes możesz zautomatyzować proces tworzenia nowych kontenerów na potrzeby swojego wdrożenia, usuwania istniejących i przejęcia zasobów przez nowe kontenery.
+* **Automatyczne rozplanowywanie wykorzystania zasobów**  
```

Comment:
The whole header sounds a bit awkward due to word `rozplanowywanie`, maybe something like `Automatyczne zarządzanie dostępnymi zasobami`, because it is a resource manager.

Reply:
Fixed.